packages/lab/src/slider/RangeSlider.tsx

```diff
@@ -0,0 +1,202 @@
+import {
+  type ChangeEvent,
+  type HTMLAttributes,
+  type SyntheticEvent,
+  forwardRef,
+} from "react";
+
+import { useControlled, useFormFieldProps } from "@salt-ds/core";
+import clsx from "clsx";
+import { SliderThumb } from "./internal/SliderThumb";
+import { SliderTrack } from "./internal/SliderTrack";
+import { useSliderThumb } from "./internal/useSliderThumb";
+import { toFloat } from "./internal/utils";
+
+export interface RangeSliderProps
+  extends Omit<HTMLAttributes<HTMLDivElement>, "onChange" | "defaultValue"> {
+  /**
+   * Initial value of the slider
```

Comment:
I felt Initial was not how we had described other uncontrolled defaults.
I then copied this from Tabs.  As a reader of docs, good to have consistent descriptions.

The default value. Use when the component is not controlled.